pkg/cmd/edge_applications/publish/publish.go

```diff
@@ -474,6 +527,34 @@ func (cmd *PublishCmd) createDomainCdn(client *apidom.Client, ctx context.Contex
 	return domain, nil
 }
 
+func (cmd *PublishCmd) createDomainStatic(client *apidom.Client, ctx context.Context, conf *contracts.AzionApplicationStatic, name string) (apidom.DomainResponse, error) {
```

Comment:
cant we use the other functions that already exist?

Reply:
Yes we could, I ended up duplicating it because it is a different structure but now it is using the same